{"uuid": "748960b9-01ff-4205-8bbb-8377ea7645af", "children": ["908545f4-3d80-4ca9-ac9f-1e09a0e04b6f"], "befores": [{"name": "update_food_init", "status": "broken", "statusDetails": {"message": "IndexError: list index out of range\n", "trace": "  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\pluggy\\_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\_pytest\\fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"c:\\users\\administrator\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\_pytest\\fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"C:\\DeliverySystem完整代码\\testCase\\conftest.py\", line 72, in update_food_init\n    idMenu = food_init.query({\"page\": 1, \"limit\": 1})['data']['records'][0]['category_id']\n"}, "start": 1704606247703, "stop": 1704606248103}], "start": 1704606247703, "stop": 1704606248117}